tasks/pocket_sand.py

```diff
@@ -0,0 +1,33 @@
+import os
+import random
+from discord_webhook import DiscordWebhook, DiscordEmbed
+import asyncio
+from tasks.base import BaseTask
+
+
+class DaleAttack(BaseTask):
+    """
+
+    When running, Dale Gibble randomly attacks with the person on thier turn, player or not
+
+    :param: None
+    :return: Embeded Discord Webhook
+    """
+
+    id = 'dale-attack'
+    channel = None
+
+    async def task(self):
+        webhook = DiscordWebhook(
+            url='https://discordapp.com/api/webhooks/596806839136419859/QtKJScaEKMfaTs2SMJVpBtuR610RtQVIUtunfgzzq3QBPw91-bLfry37lgKAlC710svS')
```

Comment:
Not sure webhooks should be checked into git. They're basically direct links to post in a channel. We might want to look at making this an environment variable if we actually need to use the webhook.

But I don't think we actually need to be using a webhook here. The bot already has the ability to post to different channels using the normal bot gateway. no need for a separate webhook